.gitignore

```diff
@@ -53,3 +53,5 @@ GRTAGS
 GSYMS
 GTAGS
 .mypy_cache/
+# TODO(#7152): remove this after the python 3 migration!
+/.pants-python-version
```

Comment:
Think the `/` isn't supposed to be there. I'm surprised the file isn't being tracked as is.